privacyidea/lib/eventhandler/scripthandler.py

```diff
@@ -80,6 +89,19 @@ def actions(cls):
         actions = {}
         for script in scripts:
             actions[script] = {
+                "background": {
+                    "type": "str",
+                    "required": True,
+                    "description": _("Wait for script to complete or run script in background. This will "
+                                     "either return the HTTP request early or could also block the request."),
+                    "value": [SCRIPT_BACKGROUND, SCRIPT_WAIT]
```

Comment:
What do you think about changing the values of ``SCRIPT_BACKGROUND`` and ``SCRIPT_WAIT`` to "background" and "wait", respectively? We cannot localize the phrases "Run in background" and "Wait for script to finish" anyway, which would look weird in German.

Reply:
ok